scripts/tff_generate_trx_from_scratch.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Generate TRX file from a collection of CSV, TXT or NPY files by individually
+specifiying positions, offsets, data_per_vertex, data_per_streamlines,
+groups and data_per_group. Each file must have its data type specified by the
+users.
+
+A reference file must be provided (NIFTI) and the option --verify_invalid will
+remove invalid streamlines (outside of the bounding box in VOX space).
+
+All dimensions (nbr_vertices and nbr_streamlines) and groups/dpg must match
+otherwise the script will (likely) crash.
+
+Each instance of --dps, --dpv, --groups require 2 arguments (FILE, DTYPE).
+--dpg requires 3 arguments (GROUP, FILE, DTYPE).
+The choice of DTYPE are:
+    - (u)int8, (u)int16, (u)int32, (u)int64
+    - float16, float32, float64
+    - bool
+
+Example command:
+tff_generate_trx_from_scratch.py fa.nii.gz generated.trx -f
+    --positions test_npy/positions.npy --positions_dtype float16 
+    --offsets test_npy/offsets.npy --offsets_dtype uint32 
+    --dpv test_npy/dpv_cx.npy uint8
+    --dpv test_npy/dpv_cy.npy uint8
+    --dpv test_npy/dpv_cz.npy uint8  
+    --dps test_npy/dps_algo.npy uint8 
+    --dps test_npy/dps_cw.npy float64 
+    --groups test_npy/g_AF_L.npy int32 
+    --groups test_npy/g_AF_R.npy int32 
+    --dpg g_AF_L test_npy/dpg_AF_L_mean_fa.npy float32 
+    --dpg g_AF_R test_npy/dpg_AF_R_mean_fa.npy float32 
+    --dpg g_AF_L test_npy/dpg_AF_L_volume.npy float32 
```

Comment:
This needs `\` for the line-breaks, right?

```suggestion
tff_generate_trx_from_scratch.py fa.nii.gz generated.trx -f \
    --positions test_npy/positions.npy --positions_dtype float16 \
    --offsets test_npy/offsets.npy --offsets_dtype uint32 \
    --dpv test_npy/dpv_cx.npy uint8 \
    --dpv test_npy/dpv_cy.npy uint8 \
    --dpv test_npy/dpv_cz.npy uint8  \
    --dps test_npy/dps_algo.npy uint8 \ 
    --dps test_npy/dps_cw.npy float64  \
    --groups test_npy/g_AF_L.npy int32 \
    --groups test_npy/g_AF_R.npy int32 \
    --dpg g_AF_L test_npy/dpg_AF_L_mean_fa.npy float32 \
    --dpg g_AF_R test_npy/dpg_AF_R_mean_fa.npy float32 \
    --dpg g_AF_L test_npy/dpg_AF_L_volume.npy float32 
```